engine/samples/hello-cubos/main.cpp

```diff
@@ -0,0 +1,102 @@
+/// [Include Components]
+#include "components.hpp"
+/// [Include Components]
+
+/// [Include]
+#include <cubos/engine/cubos.hpp>
+
+using cubos::engine::Cubos;
+/// [Include]
+
+/// [Include Entity Stuff]
+using cubos::core::ecs::Commands;
+using cubos::core::ecs::Query;
+using cubos::core::ecs::Read;
+using cubos::core::ecs::Write;
+/// [Include Entity Stuff]
+
+/// [Resource Decl]
+struct Pop
+{
+    int count;
+};
+/// [Resource Decl]
+
+/// [Hello Cubos]
+static void sayHelloCubos()
+{
+    CUBOS_INFO("Hello CUBOS");
+}
+/// [Hello Cubos]
+
+/// [Hello World]
+static void sayHello()
+{
+    CUBOS_INFO("Hello");
+}
+
+static void sayWorld()
+{
+    CUBOS_INFO("World");
+}
+/// [Hello World]
+
+/// [Entity Spawn]
+static void spawnEntities(Commands cmds, Write<Pop> pop)
+{
+    for (int i = 0; i < 10; i++)
+    {
+        cmds.create(Num{i});
+        pop->count += 1;
+    }
+}
+/// [Entity Spawn]
+
+/// [Entity Print]
+static void checkEntities(Query<Read<Num>> query, Read<Pop> pop)
+{
+    for (auto [entity, num] : query)
+    {
+        CUBOS_INFO("Entity '{}' of '{}'", num->value, pop->count);
+    }
+}
+/// [Entity Print]
+
+/// [Engine]
+int main()
+{
+    Cubos cubos{};
+    /// [Engine]
+
+    /// [Tags]
+    cubos.tag("helloTag").before("worldTag");
+    /// [Tags]
+
+    /// [Set Startup]
+    cubos.startupSystem(sayHelloCubos);
+    /// [Set Startup]
+
+    /// [Set Systems]
+    cubos.system(sayHello).tagged("helloTag");
+    cubos.system(sayWorld).tagged("worldTag");
+    /// [Set Systems]
+
+    /// [Component Add]
+    cubos.addComponent<Num>();
+    /// [Component Add]
+
+    /// [Resource Add]
+    cubos.addResource<Pop>();
+    /// [Resource Add]
+
+    /// [Entity System]
+    cubos.startupSystem(spawnEntities);
+
+    cubos.tag("entityCheckUp").after("worldTag");
```

Comment:
```suggestion
    cubos.tag("entityCheckUp");
```

Unnecessary dependency.